orc8r/cloud/go/mproto/mocks/mproto_mock.proto

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+
+package magma.orc8r.mproto_mock;
```

Comment:
Actually let's do `mproto/test/test.proto` -- I know I said "mock" originally, but really this isn't "mock" code/data, so I think test is better 🙂